README.md

```diff
@@ -1,5 +1,26 @@
-# React, Redux, Thunks & Styled Components revision
+# Europeanstar
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/adrianblynch/europeanstar.svg)](https://greenkeeper.io/)
 
-A single page app for viewing Eurostar trains to revise various techs.
+A single page app for viewing Eurostar trains.
+
+## Technologies
+
+### Frontend
+- *React* via *Create React App*
+- *Jest* for test...ing
+- *Redux* for state management
+- *Redux Thunk* for handling async actions
+- *Styled Components* because 💅
+
+### Backend
+- *Koa* serves the assets
+- Assets are compressed to *GZip* and *Brotli* (requires > Node 11.7.0)
+
+### Development
+- *GitHub* hosts the repo
+- *CircleCI* for continuous integration 
+- *Heroku* hosts the site
+- *Rollbar* tracks browser errors (at least until the demo runs out!)
+- Repo notifications sent to *Slack*
+- *Greenkeeper* checks for package updates
```

Comment:
Could add Prettier to the list

Reply:
Also, what about code coverage stats?

Reply:
Might also be an idea to add a todo section to remind yourself what else could be added

Reply:
Good idea!

Reply:
Seems Slack notifications are working now. Can we stop talking?